burger de seigneur (#17)
ShutdownRepo/The-Hacker-Recipes: physical/super-secret-zones/burger-de-seigneur.md

# 🍔 Burger de seigneur

> ##

Reviewer: ```suggestion

```

## Theory

El famoso "Burger de seigneur" is the reward meal after successfully completing your pentest. This meal was created in "montceau-les-mines" in 2022 by famous french head cook "Toma".&#x20;

> The requirements for this meal are as follow (for 3 attacker):

Reviewer: ```suggestion
The requirements for this meal are as follow (for 3 attackers):
```

* &#x20;32 slices of bacon (320 g) 🥓
* &#x20;24 slices of cheddar [🧀](https://emojipedia.org/cheese-wedge/)
* &#x20;6 slices of lard
* 600 g of minced meat&#x20;
* 3 tablespoon of  flour
* 25 g of butter 🧈
* 750 ml of semi-skimmed milk 🥛
* 3 hamburger bun
* 1 red onion 🧅

> * sauces (ketchup/ buger/ mustard)

Reviewer: ```suggestion
* sauces (ketchup/burger/mustard)
```

* 1 tomato 🍅

> * 1.6 KG french fries 🍟

Reviewer: ```suggestion
* 1.6 kg of french fries 🍟
```

> * Une bonne grose dose de talent

Reviewer: ```suggestion
* Une bonne grosse dose de talent
```

{% hint style="danger" %}
This meal can overflow your calorie counter in this case you can make your shopping by bike.&#x20;
{% endhint %}

## Practice

You need to follow this recipe step by step as follows:

### Bacon and french fries

> Preheat the oven at 220 C° and place the french fries and bacon in the oven as shown below

Reviewer: ```suggestion
Preheat the oven at 220 °C and place the french fries and bacon in the oven as shown below
```

![](<../../.gitbook/assets/frite_bacon_seigneur.png>)

### Cheddar Bechamel aka "la sauce de gros sac"

Melt the butter in a saucepan. Stir in the flour and cook, stirring constantly, until the paste cooks and bubbles a bit, but don't let it brown.&#x20;

![](<../../.gitbook/assets/bechamel2_seigneur.png>)

{% hint style="info" %}
It has to look a bit like dough
{% endhint %}

Now you can add the milk, continuing to stir as the sauce thickens. When all the milk was add you need to put 18 slice of cheddar inside.

### Create Steak

You need to create steak you have two methods&#x20;

{% tabs %}
{% tab title="Classic" %}
Take 200g of minced meat and create steaks of 200g.
{% endtab %}

{% tab title="steak de seigneur" %}
Take a salad bowl and add $nb\_attacker \* 200 g of meat as well as ketchup, minced red onion, cayenne pepper, provencal herbs.

![](<../../.gitbook/assets/steak_seigneur.png>)

> knead the mixture and create steaks of 200g.

Reviewer: ```suggestion
Knead the mixture and create steaks of 200g.
```

{% endtab %}
{% endtabs %}

### Cook the steak

> put the steak and the lard in the pan and cook it when it almost finish put 2 slice of cheddar on each steak as shown below.

Reviewer: ```suggestion
Put the steak and the lard in the pan and cook it when its almost finished put 2 slices of cheddar on each steak as shown below.
```

![](<../../.gitbook/assets/cuisson_seigneur.png>)

### Merge it together&#x20;

> Take the bread and apply a little sauce as well as a slice of tomato, steak and the lard as shown below.\

Reviewer: ```suggestion
Take the bread and apply a little sauce as well as a slice of tomato, steak and the lard as shown below.
```
